packages/core/src/components/chip/chip.scss

```diff
@@ -1,3 +1,5 @@
+@import '~@atomium/scss-utils/index';
```

Comment:
question: why do we need this sccs utility import?

Reply:
https://github.com/juntossomosmais/atomium/pull/385#discussion_r1499701612